utils/apiPluginRepos.js

```diff
@@ -0,0 +1,40 @@
+export default {
```

Comment:
I think this data should come directly from the Reaction project, not hardcoded here

Reply:
We can dynamically get the list of plugins from the `plugin.json` indeed. The concern I had is we can't map the plugin name with the repository, although we can generate  the URL in the format:
`https://github.com/reactioncommerce/${plugin-name}.git`

Reply:
Yeah, I think that naming standard should work for all the api-plugins